regex-assembly/933100.ra

```diff
@@ -0,0 +1,26 @@
+##! Please refer to the documentation at
+##! https://coreruleset.org/docs/development/regex_assembly/.
+
```

Comment:
I'd probably through in an ignore case flag for good measure, PHP tags can be uppercase.

Reply:
Thank you. Resolved.